TrackingTools/GsfTools/plugins/BuildFile.xml

```diff
@@ -1,3 +1,4 @@
+<flags   CXXFLAGS="-Ofast"/>
```

Comment:
On 7/26/16 10:42 AM, Vincenzo Innocente wrote:
> In TrackingTools/GsfTools/plugins/BuildFile.xml
> <https://github.com/cms-sw/cmssw/pull/15279#discussion_r72300077>:
> 
>> @@ -1,3 +1,4 @@
>> +<flags   CXXFLAGS="-Ofast"/>
> 
> this is a major component of the speedup...


There is essentially no speedup based on tests in HLT with PU40 done by
Andrea.


(also, recall another thread we had on -Ofast in April: there is no
significant speedup from this flag
in CMSSW RECO application
at least on the ~ 3yo  Intel and AMD hardware I have tested)

> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/15279/files/ba717540b0163453d6beeec0f7d2cf0fd87765b8#r72300077>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEdcbmcKCPhLRfTExfF15PI8TYBW-m_Zks5qZkcYgaJpZM4JU68_>.
>